supporter-product-data/src/main/scala/com/gu/conf/ZuoraQuerierConfig.scala

```diff
@@ -0,0 +1,60 @@
+package com.gu.conf
+
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagementAsyncClientBuilder
+import com.amazonaws.services.simplesystemsmanagement.model.{GetParametersByPathRequest, Parameter}
+import com.gu.aws.{AwsAsync, CredentialsProvider}
+import com.gu.model.Stage
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.collection.JavaConverters._
+import scala.concurrent.Future
+
+case class ZuoraQuerierConfig(
+  url: String,
+  partnerId: String,
+  username: String,
+  password: String,
+  discountProductRatePlanIds: List[String]
```

Comment:
since we have a closed set of stages, we can just put the non-private information e.g. the rate plans in a Map which would make the config simpler.  I think the more stuff that goes through the normal CI/CD process the better.  Also we get type checking for free and it's easier to follow.

Reply:
I did it that way initially but then you end up passing the stage value around as well as the config which is already stage specific so it's a bit more messy

Reply:
I just want to get rid of the AWS bit, it could be lookup from a Map in this piece of code if that makes more sense structurally, or even just put the stage inside the Config object if you prefer.  Either way it'll get all the benefits I mentioned, the only downside is a slight inconsistency of where the items come from.

Reply:
Done